README.md

```diff
@@ -0,0 +1,115 @@
+
+```shell
+> wget https://dl.influxdata.com/telegraf/releases/telegraf-<<VERSION>>_windows_amd64.zip -UseBasicParsing -OutFile telegraf-_windows_amd64.zip
+> Expand-Archive .\telegraf-<<VERSION>>_windows_amd64.zip -DestinationPath 'C:\Program Files\InfluxData\telegraf\'
+```
+
+Replace `<<VERSION>>` with the required version number.
+
+![Clone Repo](/img/step5clone.png)
+
+2. Unzip `telegraf.zip`
+
+![Unzip](/img/step6unzip.png)
+
+### Use WiX toolset
+
+1. Go to `Program Files` and open the WiX folder `Wix Toolset v3.11`, then open `bin` folder
+
+![WiX folder](/img/7listof.png)
+
+We need 2 files: `candle.exe` and `light.exe`
+
+2. Open command line and go to your project folder
+
+![Commandline](/img/7goto.png)
+
+3. Drag `candle.exe` into the command line to generate the path to the `candle.exe`
+
+At the end of the generated path, add:
+
+`-nologo telegraf.wxs -out telegraf.wixobj` and press `Enter`
+
+![Clone Repo](/img/9drag-candle.png)
+
+5. A new file `telegraf.wixobj` will appear in your project folder
+
+![Run candle.exe](/img/9finish.png)
+
+6. Drag `light.exe` into command line to generate the path to the `light.exe`
+
+At the end of the generated path, add:
+
+`-nologo telegraf.wixobj -out telegraf.msi` and press `Enter`
+
+![Run light.exe](/img/10start.png)
+
+7. A new file `telegraf.msi` will appear in your project folder
+
+![Run light.exe](/img/10finish.png)
+
+MSI installer is now ready.
+
+## Install Telegraf Collector and send metrics to Logz.io.
+
+That msi installer only runs in **_silent mode_**.
+
+Run the following command:
+
+```
+msiexec /i telegraf.msi LOGZIOLISTENER="<<LOGZIO_LISTENER>>" LOGZIOTOKEN="<<LOGZIO_TOKEN>>" INSTALLDIR="<<INSTALL_DIRECTION>>"
+```
+
+Replace the variables indicated between `<<>>` as per the table below:
+
+##### Parameters
+
+| Parameter         | Description                                                                                                                               |
+| ----------------- | ----------------------------------------------------------------------------------------------------------------------------------------- |
+| LOGZIO_TOKEN      | Your Logz.io account [token](<(https://app.logz.io/#/dashboard/settings/general)>).                                                       |
+| LOGZIO_LISTENER   | Logz.io [listener URL](https://docs.logz.io/user-guide/accounts/account-region.html) to ship the logs to (for example, listener.logz.io). |
+| INSTALL_DIRECTION | The folder alias of the predefined product installation folder.                                                                           |
+
+### Check Logz.io for your metrics
+
+Give your logs some time to get from your system to ours, and then open [Metrics](https://app.logz.io/#/dashboard/metrics).
+
+## Versions
+
+-   1.0.0:
+    -   Initial Release
+
+## License
+
+Licensed under the [Apache 2.0](http://apache.org/licenses/LICENSE-2.0.txt) License.
```

Comment:
please update to MIT. Telegraf agent is under MIT license and we put here an instance of it.
Also I didnt notice a license attached to this pr